website/templates/util/render_node.mako

```diff
@@ -78,7 +78,7 @@
                     <dt><span class="date log-date" data-bind="text: log.date.local, tooltip: {title: log.date.utc}"></span></dt>
                     <dd class="log-content">
                         <span data-bind="if:log.anonymous">
-                            <span><em>A user</em></span>
+                            <span data-bind="html: anonymousUserName"></span>
```

Comment:
@lyndsysimon I just tried to add `scripted`. But it doesn't work. I think it is because the class element would be shown during applying the binding. But the problem here is due to the data loading after binding.